session1/solutions/assignment3/Rakefile

```diff
@@ -1 +1,11 @@
-# assignment 3 solution
+namespace :files do
+  desc 'Rename files named *.yaml to be *.yml'
+  task :rename_yml, [:path] do |t, args|
+    path = args.path || Dir.pwd
+    files = Dir.glob(path + "/*.{yaml}").each do |f|
+      prefix = f.split("/").last
+      prefix = prefix.split(".").first
+      File.rename("#{f}", "#{path}/#{prefix}.yml")
+    end
+  end
+end
```

Comment:
Overall very good and neat implementation. Consider adding a `puts` in the end that tells the user that the file x has changed from x.yaml to x.yml